pkgs/development/python-modules/django-huey-monitor/default.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-scm,
+  django,
+  huey,
+  bx-django-utils,
+  bx-py-utils,
+  pytestCheckHook,
+  pytest-django,
+  model-bakery,
+  packaging,
+  typeguard,
+  colorlog,
+  manage-django-project,
+  pkgs,
+}:
+
+buildPythonPackage rec {
+  pname = "django-huey-monitor";
+  version = "0.9.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "boxine";
+    repo = "django-huey-monitor";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-U5Nm5YdrxmVBovYCLCEim5x8C4VuE+/J+5M5JT6upc0=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail "/bin/bash" "${lib.getExe pkgs.bash}"
+  '';
+
+  build-system = [
+    setuptools-scm
+  ];
+
+  dependencies = [
+    django
+    bx-django-utils
+    bx-py-utils
+    huey
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-django
+    model-bakery
+    packaging
+    typeguard
+    colorlog
+    manage-django-project
+    pkgs.gnumake
```

Comment:
this should be already included in the stdenv
```suggestion
```

Reply:
Also removed